client/src/components/page/CustomCallback/index.tsx

```diff
@@ -49,10 +49,11 @@ export const BaseSuccessfulLoginComponent: React.FC<RouteComponentProps> = (
     const searchString = trimStart(props.location.search, '?');
     const searchParams = querystring.parse(searchString);
     const nextPath = searchParams.next as string | undefined;
+
     if (nextPath) {
       pathToRedirectTo = nextPath;
     }
-    if (nextPath === '/') {
+    if (!nextPath || nextPath === '/') {
```

Comment:
@kelvin-muchiri just want to mention why this was.
to avoid having to show the welcome back message everytime you refreshed the page.